internal/cli/configuration.go

```diff
@@ -184,7 +198,40 @@ func ConfigurationCreate(cmd *cobra.Command, args []string) error {
 		return errors.Wrap(err, "error initializing cli")
 	}
 
-	err = client.CreateConfiguration(args[0], args[1:])
+	kvFromFiles, err := cmd.Flags().GetStringSlice("from-file")
+	if err != nil {
+		return errors.Wrap(err, "failed to read option --from-file")
+	}
+
+	// regular key/value assignments from the command line
+	kvDict := args[1:]
+
+	// extend with key/value assignments from files (--from-file option).
+	if len(kvFromFiles) > 0 {
+		for _, spec := range kvFromFiles {
+			pieces := strings.SplitN(spec, "=", 2)
+			var key string
+			var valuefile string
+			if len(pieces) < 2 {
+				// Argument is path only. Set key as last element of that path
+				valuefile = pieces[0]
+				_, key = path.Split(valuefile)
+			} else {
+				// Argument is key=path
+				key = pieces[0]
+				valuefile = pieces[1]
+			}
+			// Read file content ...
+			content, err := os.ReadFile(valuefile)
+			if err != nil {
+				return errors.Wrapf(err, "filesystem error")
+			}
+			// ... and add to command line
+			kvDict = append(kvDict, key, string(content))
+		}
```

Comment:
Based on the other comment this should be something like:

```go
fileDict := []string{}
for _, spec := range kvFromFiles {
	if strings.Contains(spec, "=") {
		spec = spec + "=" spec
	}
	fileDict = append(fileDict, spec)
}
```

Reply:
Moot with commit 03d2e28d4e39abaaf263ec52671a7e998712d6d3